README.md

```diff
@@ -1,5 +1,13 @@
 # VGA Simulator
 
+**Note:** This project is not actively maintained. Consider using one of the following newer ports:
```

Comment:
It's true that I'm not actively developing new features but it still just works.

An improvement would probably be to host it on GitHub pages though -> https://github.com/MadLittleMods/vga-simulator/issues/1

Reply:
I agree. I'll remove this comment. GH Pages would probably work better, and is free AFAIK.